Source/MainWindow.cpp

```diff
@@ -46,8 +46,8 @@ MainWindow::_PrepareMenuBar(void)
 		B_TRANSLATE("Refresh"), new BMessage(kUpdateMessage), 'R'));
 	menu->AddSeparatorItem();
 	//	Remove menu item until Deskbar replicant is fixed
-	//	menu->AddItem(fReplicantMenuItem = new BMenuItem(B_TRANSLATE("Deskbar
-	//Replicant"), 		new BMessage(kToggleDeskbarReplicantMessage), 'T'));
+	menu->AddItem(fReplicantMenuItem = new BMenuItem(B_TRANSLATE("Deskbar Replicant"),
```

Comment:
I think we generally don't have the "replicant" capitalized, see e.g. in the Deskbar "Show replicants", so:

```suggestion
	menu->AddItem(fReplicantMenuItem = new BMenuItem(B_TRANSLATE("Deskbar replicant"),
```